src/content/docs/infrastructure/host-integrations/host-integrations-list/postgresql/postgresql-integration.mdx

```diff
@@ -32,7 +32,7 @@ To install the PostgreSQL monitoring integration, you must run through the follo
 
 ### PostgreSQL versions [#postgresql-versions]
 
-Our integration is compatible with PostgreSQL until version 16.
+Our integration is compatible with PostgreSQL 17 and lower.
```

Comment:
Suggestion: Can we update this to Our integrations is compatible with PostgreSQL v17 and lower.
Reason for the suggestion: To maintain the consistency of the document across.

Reply:
Made this change.